atlas/packages/components/src/components/VideoPreview/VideoPreview.tsx

```diff
@@ -1,47 +1,86 @@
-import React from "react"
+import React from "react";
 
-import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles"
-import Avatar from "./../Avatar"
+import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles";
+import Avatar from "./../Avatar";
 
 type VideoPreviewProps = {
-  title: string
-  channel?: string
-  channelImg?: string
-  showChannel?: boolean
-  poster?: string
-  onClick?: any
-  onChannelClick?: any
-} & VideoPreviewStyleProps
+	title: string;
+	channel?: string;
+	channelImg?: string;
+	showChannel?: boolean;
+	showMeta?: boolean;
+	time?: string;
+	views?: string;
```

Comment:
Shouldn't that be a `number` instead?

Reply:
Same as above, label this as a string to skip all number processing